app/controllers/team_players_controller.rb

```diff
@@ -0,0 +1,19 @@
+class TeamPlayersController < ApplicationController
+  before_action :authorize_admin!
```

Comment:
Just realized that TeamPlayers is not a model. So, access-granted is still possible but not that simple. Then just keep it like that ;-)